tools/raw_to_pronto_code.py

```diff
@@ -25,30 +25,34 @@ def parse_and_report(rawdata_str, hertz=38000, end_usecs=100000,
   result = ["0000"]
   # Work out the frequency code.
   pronto_freq = int(1000000.0 / (hertz * 0.241246))
+  # pylint: disable=C0209
   if verbose:
-    output.write("Pronto frequency is %X (%d Hz).\n" % (pronto_freq, hertz))
+    output.write(f"Pronto frequency is %X ({hertz} Hz).\n" % pronto_freq)
```

Comment:
```suggestion
    output.write(f"Pronto frequency is {pronto_freq:X} ({hertz} Hz).\n")
```

Reply:
Awesome. I was wondering how you applied `printf` style formating of numbers etc.
That helps a lot!